lib/Models/workout_data.dart

```diff
@@ -72,13 +72,13 @@ class WorkoutData {
               newExerciseList.add(newexercise)
             });
       }
-      loaded_workout.exercises = newExerciseList;
+      loadedWorkout.exercises = newExerciseList;
       //workout = loaded_workout;
-    } catch (Exception) {
+    } catch (e) {
```

Comment:
Is there a reason you sometimes write exception and sometimes e? If not you should probably keep it consistent.

Reply:
ok i replaced catch(e) to catch(exception)